docs/cicd/jenkins.md

```diff
@@ -1,14 +1,16 @@
-# CI/CD pipelines
-The capabilities of FFC are underpinned by DevOps CI/CD pipelines providing cloud agnostic tooling with build and release automation  through Jenkins.
+# Jenkins
+Jenkins is used for all CI/CD pipelines across all FFC teams.
 
 ## Standards
-- Pipelines should be scripted within a Jenkinsfile and source controlled with the service it relates to
-- The Groovy syntax should be used
-- Build steps should be containerised where possible to reduce the plugin and Jenkins configuration dependencies
+- pipelines should be scripted within a Jenkinsfile and source controlled with the service it relates to
```

Comment:
A bit picky, but this might scan better if written more like `pipelines should be scripted within a Jenkinsfile and source controlled within the related service`

Reply:
Sounds good to me.  Changed.